tests/test_bar.py

```diff
@@ -120,11 +122,12 @@ def test_df_and_err_from_u_kln_approximates_exact_result(sigma):
 
 @pytest.mark.parametrize("sigma", [0.3, 1.0, 10.0])
 def test_df_and_err_from_u_kln_consistent_with_pymbar_bar(sigma):
-    """Compare the estimator used for 2-state delta fs (currently MBAR) with pymbar.BAR as reference."""
+    """Compare the estimator used for 2-state delta fs (currently MBAR) with pymbar.bar as reference."""
     u_kln, _ = make_gaussian_ukln_example((0.0, 1.0), (1.0, sigma))
     w_F, w_R = works_from_ukln(u_kln)
 
-    df_ref, df_err_ref = pymbar.BAR(w_F, w_R)
+    bar = pymbar.bar(w_F, w_R)
+    df_ref, df_err_ref = bar[DG_KEY], bar[DG_ERR_KEY]
```

Comment:
nit, style opinion: it feels slightly tidier to me to wrap `pymbar.bar` in `timemachine.fe.bar` as a function returning `(dg, dg_err)`, since that is the most frequent use case (avoiding the need to import `DG_KEY`, `DG_ERR_KEY`).

Reply:
updated in https://github.com/proteneer/timemachine/pull/1492/commits/cb46294fd0c28d33d45cb8b4eed06883edcee0db